common/config/rush/pnpm-lock.yaml

```diff
@@ -153,6 +153,14 @@ packages:
     dev: false
     resolution:
       integrity: sha512-RVG1Ad3Afv9gwFFmpeCXQAm+Sa0L8KEZRJJAAZEGoYDb6EoO1iQDVmoBz720h8mdrGpi0D60xNU/KhriIwuZfQ==
+  /@azure/arm-storage/16.0.0:
```

Comment:
why this change

Reply:
https://github.com/Azure/azure-sdk-for-js/issues/16208

Reply:
updated